### PR TITLE
Change order of the PropertyBuilder, so `message` is the first one in the sequence

### PR DESCRIPTION
**Pull request by hegyibalint**

### CODE REVIEW

File: subprojects/execution/src/main/java/org/gradle/internal/execution/model/annotations/InputPropertyAnnotationHandler.java [hunk 1]
```diff
@@ -92,9 +92,9 @@ private void validateNotFileType(PropertyMetadata propertyMetadata, TypeValidati
             validationContext.visitPropertyProblem(problem ->
                 problem
                     .forProperty(propertyMetadata.getPropertyName())
+                    .message(String.format("has @Input annotation used on property of type '%s'", ModelType.of(valueType).getDisplayName()))
```
Comment (tresat) [inline subprojects/execution/src/main/java/org/gradle/internal/execution/model/annotations/InputPropertyAnnotationHandler.java:95]: 💭 Providing an overload of `message(String, Object...)` that would then forward to `String.format(String, Object...)` would be a nice enhancement to keep the calls to the Problem API a little cleaner.

**Review by tresat — APPROVED** [review]

File: subprojects/execution/src/main/java/org/gradle/internal/execution/model/annotations/InputPropertyAnnotationHandler.java (hunk 1 above)
Comment (hegyibalint) [reply subprojects/execution/src/main/java/org/gradle/internal/execution/model/annotations/InputPropertyAnnotationHandler.java:95]: We already have it in the description, and we will introduce this to the `message(...)` methods soon!

Comment (reinsch82) [reply subprojects/execution/src/main/java/org/gradle/internal/execution/model/annotations/InputPropertyAnnotationHandler.java:95]: :thought_balloon: 
that is actually how the method looks like

`ProblemBuilderDefiningType message(String message, Object... args);`

We could apply it here :)

**Review by reinsch82 — APPROVED** [review]

File: subprojects/execution/src/main/java/org/gradle/internal/execution/model/annotations/InputPropertyAnnotationHandler.java (hunk 1 above)
Comment (hegyibalint) [reply subprojects/execution/src/main/java/org/gradle/internal/execution/model/annotations/InputPropertyAnnotationHandler.java:95]: Ah, it's description, which is not yet formatter friendly!

**Comment by hegyibalint** [comment]
@bot-gradle test and merge

**Comment by hegyibalint** [comment]
@bot-gradle test and merge

**Pull request merged**